frontend/packages/metal3-plugin/src/components/host.tsx

```diff
@@ -3,11 +3,11 @@ import * as _ from 'lodash';
 import * as classNames from 'classnames';
 import { sortable } from '@patternfly/react-table';
 
-import { getName, getNamespace, getMachineNode } from '@console/shared';
+import { getName, getNamespace, getUid } from '@console/shared';
```

Comment:
not new, but this should be `getUID` per console naming conventions

Reply:
Will do this as a follow up PR